sentry/src/transports/ureq.rs

```diff
@@ -0,0 +1,105 @@
+use std::time::Duration;
+
+#[cfg(fdoc)]
+use ureq_ as ureq;
+// TODO: Rename to UreqAgent?
+use ureq_::{Agent as UreqClient, AgentBuilder, Proxy};
```

Comment:
why rename at all?

Reply:
For the same reason as the rename in other modules and the comment about `Client` vs `Agent`. It's all gone now.